python/dgl/graphbolt/impl/csc_sampling_graph.py

```diff
@@ -230,13 +230,18 @@ def _convert_to_sampled_subgraph(
         )
         row = C_sampled_subgraph.indices
         type_per_edge = C_sampled_subgraph.type_per_edge
+        reverse_edge_ids = C_sampled_subgraph.reverse_edge_ids
```

Comment:
change all reverse to original? reverse_edge_ids -> original_edge_ids.

Reply:
Will be resolved in another PR